### PR TITLE
Update mule-agent-api and com.googlecode.jsendnsca dependencies

### PR DESCRIPTION
**Pull request by marialuzshin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
     </modules>
 
     <properties>
-        <mule.agent.api.version>1.7.0-mule-4.x-SNAPSHOT</mule.agent.api.version>
+        <mule.agent.api.version>1.7.0-beta.3-mule-4.x</mule.agent.api.version>
```
Comment (bahui80) [inline pom.xml]: Where is this used? We are using a beta api

Comment (aljrubior) [reply pom.xml]: Can we try to use v2.4.6 ?

Comment (aljrubior) [reply pom.xml]: The artifact version "mule-agent-modules" is  
`<version>1.7.0-mule-4.x-SNAPSHOT</version>`

It make sense to update this version?

Comment (marialuzshin) [reply pom.xml]: This dependency is 3 years old, with latest version 2.4.5 protect does not compile, 1.7.0-beta.3-mule-4.x is the most similar version founded that does not require changes.

We can upload version to 2.4.5 but we would need to make several changes and not only change the dependency.

Comment (aljrubior) [reply pom.xml]: Thanks!

**Pull request closed without merging**